profiles/package.mask

```diff
@@ -33,6 +33,134 @@
 
 #--- END OF EXAMPLES ---
 
+# hololeap <hololeap@protonmail.com> (2022-08-12)
+# Monolithic mask for dev-haskell/* packages which have no reverse dependencies,
+# are broken, or severely out of date. The aim is to have the Haskell overlay
+# (::haskell) be the place for development packages and only have packages
+# needed for end-user applications in ::gentoo, as the status quo has
+# proven to be unsustainable. More up-to-date versions of these packages
+# are available in ::haskell.
+# Removal on 2022-09-12.
+app-portage/fquery
+dev-games/goatee
+dev-haskell/megaparsec
+dev-haskell/parser-combinators
+dev-haskell/megaparsec-tests
+dev-haskell/hspec-megaparsec
+dev-haskell/neat-interpolation
+dev-haskell/stack
```

Comment:
Users have been complaining about stack not working in the bug tracker. Is it worth keeping? I personally don't see why but some upstreams use it for development.